pkgs/development/python-modules/craft-application/default.nix

```diff
@@ -39,12 +39,12 @@ buildPythonPackage rec {
     owner = "canonical";
     repo = "craft-application";
     tag = version;
-    hash = "sha256-rKeEi9z5eQfjn0Q4FR6CVIz9YDS7Ejg4cqcor5wtz0s=";
+    hash = "sha256-DprItAuGjw8AACeJDrIa6KIWLSyImuWI0qeROpPohtM=";
   };
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace-fail "setuptools==75.2.0" "setuptools"
+      --replace-fail "setuptools==75.8.0" "setuptools"
```

Comment:
Can't we use `pythonRelaxDeps` ? (I have the feeling I already asked this question, am I crazy or... ? :D )